.travis.yml

```diff
@@ -108,13 +108,19 @@ jobs:
       os: linux
       language: python
       env: [ TEST_SUITE=build, 'SPEC=mpich' ]
+    - stage: 'docker build'
+      sudo: required
+      os: linux
+      language: generic
+      env: TEST_SUITE=docker
   allow_failures:
     - dist: xenial
 
 stages:
   - 'style checks'
   - 'unit tests + documentation'
   - 'build tests'
+  - 'docker build'
```

Comment:
Does this need to run every time, on every PR?  It seems like that's too frequent.  I think it should only run on branch builds for develop -- so when `TRAVIS_BRANCH` is set to `develop`.

Reply:
There is also a new `if` condition: https://blog.travis-ci.com/2017-09-12-build-stages-order-and-conditions
```
  - name: 'docker build'
    if: branch = develop```